entries/t/test.de.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Stiftung Warentest": {
+    "domain": "test.de",
+    "url": "https://www.test.de/",
```

Comment:
```suggestion
```

The `url` key is only needed if going to the domain (e.g. `https://test.de`) fails